go.mod

```diff
@@ -36,41 +36,16 @@ require (
 require (
 	github.com/Microsoft/go-winio v0.5.1 // indirect
 	github.com/Microsoft/hcsshim v0.8.14 // indirect
-	github.com/cilium/ebpf v0.4.0 // indirect
```

Comment:
I added some changes to this file for some reason. (https://github.com/Yuya9786/gvisor/commit/03a4c5abf6b236d036d6074b6b18fca24c4c2cb4)
But I noticed this phenomenon since CI didn't succeed.
So I did `go mod tidy`. (https://github.com/google/gvisor/pull/7285/commits/e85c42a1a2a0c39420b1d7449cf6cb84e329589d)
But `github.com/cilium/ebpf v0.4.0` wasn't recovered.
I thought it's because there might be no longer direct and indirect reference.
So what should I do?

Reply:
My  `go mod tidy` might not have been executed correctly.
```
$ go mod tidy
go: finding module for package gvisor.dev/gvisor/pkg/sentry/control/control_go_proto
go: finding module for package gvisor.dev/gvisor/pkg/sentry/kernel/uncaught_signal_go_proto
go: finding module for package gvisor.dev/gvisor/pkg/sentry/kernel/memevent/memory_events_go_proto
go: finding module for package gvisor.dev/gvisor/pkg/metric/metric_go_proto
go: finding module for package gvisor.dev/gvisor/pkg/sentry/strace/strace_go_proto
go: finding module for package gvisor.dev/gvisor/pkg/sentry/arch/registers_go_proto
go: finding module for package gvisor.dev/gvisor/pkg/eventchannel/eventchannel_go_proto
go: finding module for package gvisor.dev/gvisor/pkg/sentry/unimpl/unimplemented_syscall_go_proto
go: finding module for package gvisor.dev/gvisor/test/packetimpact/proto/posix_server_go_proto
gvisor.dev/gvisor/pkg/eventchannel imports
        gvisor.dev/gvisor/pkg/eventchannel/eventchannel_go_proto: no matching versions for query "latest"
gvisor.dev/gvisor/pkg/metric imports
        gvisor.dev/gvisor/pkg/metric/metric_go_proto: no matching versions for query "latest"
gvisor.dev/gvisor/pkg/sentry/arch imports
        gvisor.dev/gvisor/pkg/sentry/arch/registers_go_proto: no matching versions for query "latest"
gvisor.dev/gvisor/pkg/sentry/kernel imports
        gvisor.dev/gvisor/pkg/sentry/kernel/uncaught_signal_go_proto: no matching versions for query "latest"
gvisor.dev/gvisor/pkg/sentry/kernel imports
        gvisor.dev/gvisor/pkg/sentry/unimpl/unimplemented_syscall_go_proto: no matching versions for query "latest"
gvisor.dev/gvisor/pkg/sentry/kernel/memevent imports
        gvisor.dev/gvisor/pkg/sentry/kernel/memevent/memory_events_go_proto: no matching versions for query "latest"
gvisor.dev/gvisor/pkg/sentry/strace imports
        gvisor.dev/gvisor/pkg/sentry/strace/strace_go_proto: no matching versions for query "latest"
gvisor.dev/gvisor/runsc/boot imports
        gvisor.dev/gvisor/pkg/sentry/control/control_go_proto: no matching versions for query "latest"
gvisor.dev/gvisor/test/packetimpact/testbench imports
        gvisor.dev/gvisor/test/packetimpact/proto/posix_server_go_proto: no matching versions for query "latest"
gvisor.dev/gvisor/tools/go_generics/tests/all_types imports
        ./lib: "./lib" is relative, but relative import paths are not supported in module mode
```
What should I do to import these packages correctly?

Reply:
bazel will update the go.mod file when building. I don't remember the exact reason. We should improve how this works at some point.

For now if you see these changes you can run
`git checkout master go.mod go.sum` to restore them to the original version. Since you've already committed the changes you'll need to make another commit to revert them.